src/testdrive/action/mod.rs

```diff
@@ -89,6 +92,7 @@ pub trait Action {
 pub fn build(cmds: Vec<PosCommand>, state: &State) -> Result<Vec<PosAction>, Error> {
     let mut out = Vec::new();
     let mut vars = HashMap::new();
+    let mut timeout = DEFAULT_TIMEOUT;
```

Comment:
Can we call this `sql_timeout` to make it more clear that this only applies to SQL statements?